app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -22,7 +22,7 @@
   </div>
 
   <div class="col-md-2">
-    <%= f.date_field :initial_call_date, label: "Initial Call Date", value: @today %>
+    <%= f.date_field :initial_call_date, label: "Initial Call Date", value: @today,  required: "required" %>
```

Comment:
The `required: 'required'` thing is interesting here! Is this a pattern we should use for the name and phone fields too?